## Haplotype-resolved T2T Genome Assemblies and Pangenome Graph of Pear

- Author: Qionghou Li (2019204004@njau.edu.cn)

## Introduction

This repository includes the pipeline used for the research presented in the article _"Haplotype-resolved T2T genome assemblies and pangenome graph of pear reveal high-resolution allele-specific expression and expanding set of genetic variations"_. The study focuses on pear genome assemblies and the construction of a pangenome graph, highlighting allele-specific expressions and a broad set of genetic variations.

## Repository Overview

The pipeline encompasses several key processes:

1. **Reads Partition**: Classifying and organizing sequencing reads based on their genomic origin.

2. **Allelle Identification**: Identification allelles from two haplotype genomes.

3. **Pangenome Construction**: Building a pangenome graph based on minigraph-cactus apporach

4. **Graph-based SNP Calling**: Utilizing the pangenome for SNP calling across different pear accessions.

5. **SV Hotspots Identification**: Detecting structural variant (SV) hotspots within the population, which are critical for understanding genetic diversity.

## License

This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details. 

## Citation

Haplotype-resolved T2T genome assemblies and pangenome graph of pear reveal high-resolution allele-specific expression and expanding set of genetic variations (Submitted)